src/app/shared/visitor-navbar/style.scss

```diff
@@ -0,0 +1,44 @@
+@use '@angular/material' as mat;
+@import '../../styles/palettes';
```

Comment:
Trzeba pozmieniać te importy na @use, Grzesiek na discordzie wysłał w jaki sposób to robić